pkg/reconciler/kubernetes/tektonconfig/extension/dashboard.go

```diff
@@ -61,10 +61,14 @@ func ensureTektonDashboardExists(clients op.TektonDashboardInterface, config *v1
 
```

Comment:
may be we should also check if an existing instance doesn't have an owner ref then update that instance

Reply:
Have updated the patch could you please take a look at it once